test/quoter.ts

```diff
@@ -0,0 +1,90 @@
+import { BigInt as ProtoBigInt } from 'blndgs-model';
+import { toBigInt } from '../src';
+import { Token } from '../test/constants';
+
+/**
+ * Represents a request for a quote from the Borsa Network API
+ */
+export interface BorsaQuoteRequest {
+  sender: string;
+  nonce: number;
+  intent: {
+    fromAsset: {
+      address: string;
+      amount: ProtoBigInt
+      chainId: ProtoBigInt
+    };
+    toAsset: {
+      address: string;
+      amount: ProtoBigInt
+      chainId: ProtoBigInt
+    };
+  };
+}
+
+/**
+ * Represents a response from the Borsa Network API
+ */
+export interface BorsaQuoteResponse {
+  steps: {
+    name: string;
+    operation_details: {
```

Comment:
the `operation_details` can be many types e.g.; `transfer`, `approval`, `swap` etc. we need typings